### ISSUE TITLE
FR: Change the Name of an Existing Organization thru the Admin Console

### ISSUE THREAD

**Issue by gpa71** [tailscale/tailscale#13162]
### What are you trying to do?

I want to be able to change the name of an existing organization through the Admin Console

### How should we solve this?

Enable this in the Admin Console

### What is the impact of not solving this?

Not having to delete the existing setup and recreate it from scratch only to change the Organization Name

**Comment by bradfitz** [comment]
Work is underway on this, but it's part of a much bigger data model change that won't be ready for several months at the earliest.

**Comment by matthewjewell** [comment]
> Work is underway on this, but it's part of a much bigger data model change that won't be ready for several months at the earliest.

@bradfitz Is this still planned?

**Comment by bradfitz** [comment]
Yes, it's rolling out now. It's available to some people, but not everybody yet.
